### ISSUE TITLE
Merging results files code update

### ISSUE THREAD

**Issue by bdemeulder** [bdemeulder/PIONEERmetastaticTreatment#6]
The code to merge results files in codeToRun.R lines 204-205 is no longer valid. 

We need to update this to work with the Andromeda SQLlite (see https://github.com/OHDSI/CohortDiagnostics/pull/721)

**Comment by Tsemharb** [comment]
Yeah, it no longer works with csv files, however, merging in Andromeda turned out to be pretty cumbersome. It works, but the code I wrote is essentially one big crutch. I didn't include this script into package intentionally. So as for now I think it should suffice to simply remove the old merge instruction
